x/accounts/defaults/feegrant/feegrant.go

```diff
@@ -0,0 +1,268 @@
+package feegrant
+
+import (
+	"context"
+	"time"
+
+	"cosmossdk.io/collections"
+	"cosmossdk.io/core/address"
+	"cosmossdk.io/core/appmodule"
+	corecontext "cosmossdk.io/core/context"
+	"cosmossdk.io/core/event"
+	errorsmod "cosmossdk.io/errors"
+	"cosmossdk.io/x/accounts/accountstd"
+	"cosmossdk.io/x/accounts/defaults/feegrant/v1"
+	"cosmossdk.io/x/accounts/internal/implementation"
+	xfeegrant "cosmossdk.io/x/feegrant"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+)
+
+type Dependencies = implementation.Dependencies
+
+var (
+	// todo: prefix conflicts when embedded in other account type. bump +100
```

Comment:
We would need some rules to reserve a range for this and other cross-cutting features in the future